generated/go/google-cloud-speech-v1beta1/cloud.google.com/go/speech/apiv1beta1/doc.go

```diff
@@ -1,33 +0,0 @@
-// Copyright 2017, Google Inc. All rights reserved.
```

Comment:
Note to reviewers: these deleted files are duplicates and can be safely deleted.